R/model.R

```diff
@@ -9,16 +9,27 @@
 #'
 #' @param timesteps the number of timesteps to run the simulation for
 #' @param parameters a named list of parameters to use
+#' @param c_param correlation parameters
 #' @export
-run_simulation <- function(timesteps, parameters = NULL) {
+run_simulation <- function(timesteps, parameters = NULL, c_param = NULL) {
```

Comment:
Yeah it's a real shame. The main issue is that `individual` parameters only supports 1D vectors at the moment. So that they can easily be translated to C++.

I should deffo add support matrix parameters like rho or contact matrices. But for now, it has to be done outside of `individual`